Documentation/operations/upgrade.rst

```diff
@@ -334,6 +338,8 @@ Removed Options
   in version 1.13.
 * The ``host-reachable-services-protos`` option (``.hostServices.protocols`` in
   Helm) was deprecated, and it will be removed in version 1.13.
+* The ``subnet-tags-filter`` option (``.eni.subnetTagsFilter`` in Helm) was deprecated
+  and replaced by the ``instance-tags-filter`` option. It will be removed in version 1.13.
```

Comment:
So I think given our conclusion in https://github.com/cilium/cilium/pull/19181/files#r838850096 I think we want to un-deprecate this, right?

Reply:
To be clear... we keep both option, BUT do we keep it as-is, where `instance filter` has precedence over `subnet filter` in `GetInstances()` ? 
(I guess we should keep it like that)

Reply:
I updated the code to remove the deprecation notice

Reply:
> To be clear... we keep both option, BUT do we keep it as-is, where instance filter has precedence over subnet filter in GetInstances() ?

Yes, perfect, thanks!